client/server/middleware/test/logger.js

```diff
@@ -78,6 +82,9 @@ it( 'Adds a `logger` property to the request', () => {
 	} );
 
 	expect( req.logger ).toBe( mockLogger );
+	expect( req.logger.child ).toHaveBeenCalledWith( {
+		reqId: '00000000-0000-0000-0000-000000000000',
+	} );
```

Comment:
Also, `req.logger` is a return value of `rootLogger.child()`, and the `.child` method of `req.logger` (i.e., creating child of a child) is never called in the real code. The test would fail on a real Bunyan logger.

Reply:
My assumption here was that if we limited the logs to a narrow period of time, chances for conflicts and non-unique IDs would be extremely minimal. Not sure we can guarantee they won't happen, though 😉